pkg/iac/scanners/helm/scanner.go

```diff
@@ -150,6 +157,7 @@ func (s *Scanner) ScanFS(ctx context.Context, target fs.FS, path string) (scan.R
 			} else {
 				results = append(results, scanResults...)
 			}
+			return fs.SkipDir
```

Comment:
For my understanding: could you explain why we return early here?

Reply:
We recursively traverse the directories looking for a Chart. If we detect a Chart by the Chart.yaml file, we must stop traversing that directory, as that Chart is further processed by the parser, otherwise Subcharts will be rescanned.